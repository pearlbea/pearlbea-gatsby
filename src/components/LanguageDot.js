import React from "react";
import dotStyle from "./dot.module.css";

export default ({ language }) => {
  const langClass = language ? language.toLowerCase() : "dot";
  const dotClass = dotStyle[langClass];

  return <div className={dotClass} />;
};
