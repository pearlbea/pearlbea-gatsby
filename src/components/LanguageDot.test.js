import React from "react";
import renderer from "react-test-renderer";
import LanguageDot from "./LanguageDot";

describe("LanguageDot", () => {
  it("should render", () => {
    const tree = renderer.create(<LanguageDot language="HTML" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
