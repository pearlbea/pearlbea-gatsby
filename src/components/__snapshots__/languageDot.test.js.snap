// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LanguageDot should render 1`] = `<languageDot />`;
